src/js/utils/bounds.js

```diff
@@ -38,6 +38,110 @@ export function getBounds(el) {
   return box;
 }
 
+/**
+ * Gets bounds for when target modifiier is 'scroll-handle'
+ * @param target
+ * @return {{left: number, width: number, height: number}}
```

Comment:
for generated docs, does this work correctly?

Reply:
Unsure, this is just what WebStorm added in automatically. Figured we would cleanup all the comments when we have the time to switch the docs to JSDoc like Shepherd.